From e938aa387f07cd7b3c608af4d11a075c67c5a11d Mon Sep 17 00:00:00 2001
From: Rimon Xu <rimon.xu@rock-chips.com>
Date: Thu, 14 Dec 2017 11:08:22 +0800
Subject: [PATCH] arm64: rockchip_defconfig: select mpp device and service

enable CONFIG_ROCKCHIP_MPP_SERVICE and CONFIG_ROCKCHIP_MPP_DEVICE
of rockchip_defconfig for encoder.

Change-Id: I30069af7af82f6731f322c4b93e3a7c6009e8434
Signed-off-by: Rimon Xu <rimon.xu@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index bda4aa268d42..1032f9fd007e 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -513,6 +513,8 @@ CONFIG_ROCKCHIP_RGA2=y
 CONFIG_RK_VCODEC=y
 CONFIG_IEP=y
 CONFIG_ROCKCHIP_DP=y
+CONFIG_ROCKCHIP_MPP_SERVICE=y
+CONFIG_ROCKCHIP_MPP_DEVICE=y
 CONFIG_LOGO=y
 # CONFIG_LOGO_LINUX_MONO is not set
 # CONFIG_LOGO_LINUX_VGA16 is not set
-- 
2.35.3

